manifest.yml

```diff
@@ -23,13 +23,13 @@ url_to_dependency_map:
 dependencies:
   - name: openresty
     version: 1.9.3.1
-    uri: https://raw.githubusercontent.com/muymoo/staticfile-buildpack/files/vendor/openresty-1.9.3.1.tar.gz
+    uri: https://raw.githubusercontent.com/sks/staticfile-buildpack/files/vendor/openresty-1.9.7.3.tar.gz
```

Comment:
I am not sure If I got this change in https://github.com/muymoo/staticfile-buildpack/pull/11